src/redux/configureStore.js

```diff
@@ -1,10 +1,18 @@
-import { configureStore } from '@reduxjs/toolkit';
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Could you avoid disabling linter checks in line because they represent coding standards you share with your team? Most of the time they also enforce best coding practices.